### ISSUE TITLE
Update create healthsite API

### ISSUE THREAD

**Issue by meomancer** [healthsites/healthsites#1119]
Update creating of healthsite API to use docker osm database
Please also handle options value that can be checked on osm

see here as reference : https://docs.google.com/spreadsheets/d/1P11VJPocQ5EkmoFaTMItg3z6Rdon2A1M2kIqqjIc-Cw/edit#gid=1195977473

child of https://github.com/healthsites/healthsites/issues/1116
depend on #1129

**Comment by meomancer** [comment]
Already done by @myarjunar 
Working with format in schema
